org.freedesktop.Sdk.Extension.openjdk11.appdata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.freedesktop.Sdk.Extension.openjdk11</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>OpenJDK 11 SDK extension</name>
```

Comment:
`project_license`?

Reply:
Okay I added licenses for everything in the source tree to be safe.

Reply:
I'd say those others should have a license too.